DayrSuryan/DSEthiop12.xml

```diff
@@ -732,13 +732,15 @@ schematypens="http://relaxng.org/ns/structure/1.0"
                <history>
                 <origin>
 
-              <origDate notBefore="1920" notAfter="1953" evidence="lettering">Around the middle of the 20th century,
-                before 1954</origDate>.
-                The parts of the manuscript could be written by the same
-                scribe in Egypt, but at different times, trimmed and bound together
-
-                 sometime around <date>1950</date>, but before <date>1954</date>).
-                  The owner and possibly the scribe was <persName ref="PRS11905GabraM">Gabra Māryām of Gondar</persName>.
+              <origDate notBefore="1840" notAfter="1870" evidence="lettering">Ca. 1840-1870</origDate>.
+              Parts of the manuscript wer written in Egypt, over a period of time,
+               later they were gathered, trimmed and bound together.
+                 The main <ref target="#h1"/> seems to be identical to
```

Comment:
I think it is better to write in the content of the refs what we want to have printed, so The `<ref target="#h1">main hand</ref>` or `<ref target="#h1">Hand 1</ref>` , etc. also for the following refs in this paragraph

Reply:
Like this (see new version)?

Reply:
perfect, I think